tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "declaration": true,
     "outDir": "lib",
     "strict": true,
-    "lib": ["dom", "es2019"],
+    "lib": ["dom", "es2022"],
```

Comment:
Does this change break compatibility for users targeting older JS versions?

Reply:
Claude tells me it's okay since this only affects type checking:

> The "lib" option in tsconfig.json specifies which built-in TypeScript type definitions your project can use. It determines which JavaScript APIs and features are available to your code. In your case:
> 
> * "dom" - Provides type definitions for browser APIs (like document, window, HTMLElement, etc.)
> * "es2022" - Provides type definitions for ECMAScript 2022 features (including Error cause, Array.at(), Object.hasOwn(), etc.)
> 
> Without specifying these libraries:
> 
> 1. TypeScript wouldn't know about browser-specific APIs (from "dom")
> 2. TypeScript wouldn't recognize newer JavaScript features (from "es2022")
> 
> For example, if you only had "es2019" instead of "es2022", the Error constructor with cause would give you a type error because that feature wasn't available until ES2022.
> 
> It's important to note that "lib" only affects TypeScript's type checking - it doesn't affect how your code is compiled. The actual JavaScript features available at runtime are determined by your "target" setting and your runtime environment.